File: reverse_string/reverseString.js


"use strict";

// function reverseString(str) {
//   // split string, reverse array, join array into string
//   let splitString = str.split('').reverse().join('');
//
//   // console.log(splitString);
//   return splitString;
//
// }

const reverseString = str => {
    // split string, reverse array, join array into string
  let splitString = str.split('').reverse().join('');

  // console.log(splitString);
  return splitString;

};

reverseString("hello");
// should return "olleh"

reverseString("Howdy");
// should return  "ydwoH"

reverseString("Greetings from Earth")
// should return "htraE morf sgniteerG"

File: reverse_string/reverse-string.py
# Reverse a string.
"""
>>> reverse_string('hello')
'olleh'

>>> reverse_string('Howdy')
'ydwoH'

>>> reverse_string('Greetings from Earth')
'htraE morf sgniteerG'

"""

def reverse_string(string: str) -> str:
    """
    Function accepts a string and reverses it.
    :param str: string
    :return: string
    """

    reversed_str = string[::-1]
    return reversed_str


